joplin/pages/official_documents_page/models.py

```diff
@@ -68,6 +72,11 @@ class OfficialDocumentPage(JanisBasePage):
         InlinePanel('official_document_collection', label="Official document collections this document belongs to"),
     ]
 
+    class Meta:
+        indexes = [
+            models.Index(fields=['-date',]),
+        ]
```

Comment:
🎊

Reply:
In my tests, adding this index only shaved off like half a millisecond. But it's still the right thing to do, and it'll help in the long run. I documented my experiment here: https://github.com/cityofaustin/techstack/issues/4657